Manifests/ManagedPreferencesApplications/com.secondsonconsulting.renew.plist

```diff
@@ -0,0 +1,487 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>pfm_app_url</key>
+	<string>https://github.com/SecondSonConsulting/Renew/releases</string>
+	
```

Comment:
Lots of empty line spaces, let's clean these up pls

Reply:
Completed